### PR TITLE
Task 2: MongoDB connectivity

### PR DESCRIPTION
**Pull request by frostbite127**
Please pull these awesome changes in!

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
-const http = require('http');
+// const http = require('http');
 
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
-  res.end('Hello World\n');
-});
+// const server = http.createServer((req, res) => {
+//   res.statusCode = 200;
```
Comment (frostbite127) [inline server.js]: Consider renaming the constant variable 'PORT' to something more descriptive to avoid confusion with the PORT variable defined earlier in the code.

Comment (frostbite127) [inline server.js]: Duplicate declaration of constant 'PORT'. Consider renaming one of them to avoid confusion.

File: server.js [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
+//   res.setHeader('Content-Type', 'text/plain');
```
Comment (frostbite127) [inline server.js]: The variable name 'mongoUri' should be more descriptive to indicate its purpose or usage.

File: server.js [hunk 3]
```diff
@@ -1,34 +1,34 @@
...
+//   res.end('Hello World\n');
```
Comment (frostbite127) [inline server.js]: Please remove the placeholder text '<mongo_cluster_url>' and provide the actual MongoDB cluster URL for security reasons.

File: server.js [hunk 4]
```diff
@@ -1,34 +1,34 @@
 // TASK 1: Setting up server
 
 
-const http = require('http');
+// const http = require('http');
 
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
```
Comment (frostbite127) [inline server.js:8]: The 'http' module is not imported in this file. You need to import the 'http' module using: const http = require('http');

File: server.js [hunk 5]
```diff
@@ -1,34 +1,34 @@
...
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
-  res.end('Hello World\n');
-});
+// const server = http.createServer((req, res) => {
+//   res.statusCode = 200;
+//   res.setHeader('Content-Type', 'text/plain');
+//   res.end('Hello World\n');
+// });
```
Comment (frostbite127) [inline server.js]: The 'mongoUri' variable seems out of place in this code snippet. It should be placed before the server creation code.

File: server.js (hunk 4 above)
Comment (frostbite127) [inline server.js:8]: The 'http' module is used in this code snippet but it is not imported. It needs to be imported at the beginning of the file using: const http = require('http');

File: server.js [hunk 6]
```diff
@@ -1,34 +1,34 @@
...
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
-  res.end('Hello World\n');
-});
 
-const PORT = 8000;
-const HOSTNAME = 'localhost';
 
-server.listen(PORT, HOSTNAME, () => {
```
Comment (frostbite127) [inline server.js:15]: The 'mongoUri' variable seems to be unrelated to the rest of the code. It should be removed or placed in an appropriate context.

File: server.js [hunk 7]
```diff
@@ -1,34 +1,30 @@
 // TASK 1: Setting up server
 
 
-const http = require('http');
+// const http = require('http');
 
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
```
Comment (frostbite127) [inline server.js:8]: Consider removing the duplicate declaration of 'PORT'. It is already declared on line 2.

File: server.js [hunk 8]
```diff
@@ -1,34 +1,30 @@
...
 
 
-const http = require('http');
+// const http = require('http');
 
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
-  res.end('Hello World\n');
-});
```
Comment (frostbite127) [inline server.js:10]: The mongoUri should be securely stored, do not hardcode sensitive information like passwords directly in the code.

Comment (frostbite127) [inline server.js:10]: The mongoUri should not include the credentials and the URL should be properly formatted to establish a connection to the MongoDB database. Including sensitive information like credentials in the code is not recommended for security reasons.

File: server.js (hunk 7 above)
Comment (frostbite127) [inline server.js:8]: Define the 'server' variable and create an HTTP server using 'http.createServer()' method before trying to listen on a port and hostname.

File: server.js [hunk 9]
```diff
@@ -1,34 +1,30 @@
 // TASK 1: Setting up server
 
 
-const http = require('http');
+// const http = require('http');
 
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
-  res.end('Hello World\n');
```
Comment (frostbite127) [inline server.js:9]: Consider defining the 'server' variable before attempting to listen on a port and hostname.

File: server.js (hunk 7 above)
Comment (ShubhamGupta24) [reply server.js:8]: > but it is already mentioned in the code

Comment (ShubhamGupta24) [reply server.js:8]: line 37

File: server.js [hunk 10]
```diff
@@ -1,35 +1,31 @@
 // TASK 1: Setting up server
 
 
-const http = require('http');
+// const http = require('http');
 
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
```
Comment (frostbite127) [inline server.js:8]: Consider removing the redeclaration of the 'PORT' variable. Since it is already defined on line 2, there is no need to redefine it on line 9.

File: server.js [hunk 11]
```diff
@@ -1,35 +1,31 @@
 // TASK 1: Setting up server
 
 
-const http = require('http');
+// const http = require('http');
 
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
-  res.end('Hello World\n');
```
Comment (frostbite127) [inline server.js:9]: Consider removing the redeclaration of the 'HOSTNAME' variable. Since it is already defined on line 3, there is no need to redefine it on line 10.

File: server.js [hunk 12]
```diff
@@ -1,35 +1,31 @@
...
 
 
-const http = require('http');
+// const http = require('http');
 
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
-  res.end('Hello World\n');
-});
```
Comment (frostbite127) [inline server.js:6]: The code is using 'MongoClient' class, but there is no import statement for 'MongoClient' in the provided code. Make sure to import 'MongoClient' before using it.

Comment (ShubhamGupta24) [reply server.js:6]: Connected to MongoDB appeared in the terminal
